doc/ubuntu-build.md

```diff
@@ -19,9 +19,10 @@
    git clone https://github.com/hyperledger/indy-sdk.git
    cd ./indy-sdk/libindy
    cargo build
+   cd ..
    ```
 1. Run integration tests:
-   * Start local nodes pool on `127.0.0.1:9701-9708` with Docker:
+   * Start local nodes pool on `127.0.0.1:9701-9709` with Docker:
```

Comment:
Actually pool use only 9701-9708 ports. So this change should not be applied.
@vimmerru I think we should fix it in appropriate dockerfile also. May be we can merge this PR, and fix this line and dockerfile in the new one.